yt_dlp/extractor/rcs.py

```diff
@@ -15,6 +15,7 @@
 
 
 class RCSBaseIE(InfoExtractor):
+    # class converted from the js player
```

Comment:
what does this mean?

Reply:
it's the javascript player used by the website that takes the manifest url in the page and changes it to get the real url and I had to do the same by making a python version of that script.

maybe the comment is not super clear... maybe I'll add the js url as a reference?

https://js2.corriereobjects.it/includes2013/LIBS/js/corriere_video.sjs
and look for `VideoPlayerLoader.prototype.transformSrc`

Reply:
```suggestion
    # based on VideoPlayerLoader.prototype.transformSrc from
    # https://js2.corriereobjects.it/includes2013/LIBS/js/corriere_video.sjs
```